source/extensions/filters/http/admission_control/admission_control.cc

```diff
@@ -123,9 +132,14 @@ AdmissionControlFilter::encodeTrailers(Http::ResponseTrailerMap& trailers) {
 
 bool AdmissionControlFilter::shouldRejectRequest() const {
   const auto request_counts = config_->getController().requestCounts();
-  const double total = request_counts.requests;
-  const double success = request_counts.successes;
-  const double probability = (total - config_->aggression() * success) / (total + 1);
+  const double total_requests = request_counts.requests;
+  const double successful_requests = request_counts.successes;
+  double probability = total_requests - successful_requests / config_->successRateThreshold();
```

Comment:
Link to docs so that people can read an explanation, or if they change it in the future they update the docs.